recipes/upgrades.rb

```diff
@@ -0,0 +1,23 @@
+# encoding: utf-8
+#
+# Cookbook Name:: nginx-hardening
+# Recipe:: default.rb
+#
+# Copyright 2015, Edmund Haselwanter
+# Copyright 2015, Deutsche Telekom AG
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# nginx requires up to date openssl packages
+include_recipe 'openssl::upgrade'
```

Comment:
if we want to be able to test it. yes. otherwise the nginx module will not install (dep resolution problem). and this is used as a common place for upgrades with only one upgrade as of today. it is used to be more visual what has to be applied. 

we could remove it if you wish and only include `openssl::upgrade` in the test run_list if you wish